okex-client/tests/client.rs

```diff
@@ -47,7 +49,7 @@ async fn client_is_missing_header() -> anyhow::Result<()> {
 
 #[tokio::test]
 async fn transfer_funding_to_trading() -> anyhow::Result<()> {
-    let amount = 0.00001;
+    let amount = Decimal::new(1, 5);
```

Comment:
lets use from `from_str` or `dec!` micro for creating hard coded decimals in tests. I think that's more readable than the `(1, 5)` format

Reply:
I had initially used `from_str`, but thought `new` was much cleaner. I'll revert.